test/integration/associations/belongs-to-many.test.js

```diff
@@ -538,8 +544,8 @@ describe(Support.getTestDialectTeaser('BelongsToMany'), function() {
     });
 
     it('should be able to set twice with custom primary keys', function() {
-      var User = this.sequelize.define('User', { uid: { type: DataTypes.INTEGER, primaryKey: true, autoIncrement: true }, username: DataTypes.STRING })
-        , Task = this.sequelize.define('Task', { tid: { type: DataTypes.INTEGER, primaryKey: true, autoIncrement: true }, title: DataTypes.STRING });
+      var User = this.sequelize.define('User', { myuid: { type: DataTypes.INTEGER, primaryKey: true, autoIncrement: true }, username: DataTypes.STRING })
+        , Task = this.sequelize.define('Task', { mytid: { type: DataTypes.INTEGER, primaryKey: true, autoIncrement: true }, title: DataTypes.STRING });
```

Comment:
Wut :-D. Isn't the column name escaped? In that case I'd think you should be able to use whatever name you want..